docs/support-bundle-guide.md

```diff
@@ -0,0 +1,247 @@
+| NetworkPolicy Resources    | A, C, O, SBC-N, SBC-E       | YAML output of running `antctl get appliedtogroups`, `antctl get addressgroups`                                                                                                                                                                                       |
+| Heap Pprof                 | A, C, O, SBC-N, SBC-E       | Result of running [`pprof.WriteHeapProfile`](https://pkg.go.dev/runtime/pprof#WriteHeapProfile)                                                                                                                                                                       |
+| HNSResources(Windows Only) | A, O, SBC-N, SBC-E          | Result of running `Get-HnsNetwork` and `Get-HNSEndpoint`                                                                                                                                                                                                              |
+| Antrea Agent Info          | A, O, SBC-N, SBC-E          | YAML output of running `antctl get agentinfo`                                                                                                                                                                                                                         |
+| Antrea Controller Info     | C, O                        | YAML output of running `antctl get controllerinfo`                                                                                                                                                                                                                    |
+| IP Address info            | A, O, SBC-N, SBC-E          | Result of running `ip address` in Linux or `ipconfig /all` in Windows                                                                                                                                                                                                 |
+| IP Route info              | A, O, SBC-N, SBC-E          | Result of running `ip route` in Linux or `route print` in Windows                                                                                                                                                                                                     |
+| IP Link info               | A, O, SBC-N, SBC-E          | Result of running `ip link` in Linux or `Get-NetAdapter` in Windows                                                                                                                                                                                                   |
+| Cluster Information        | O                           | Concatenation of information, including: 1. all Pods, Deployments, Replicasets and Daemonsets in all namespaces with any resourceVersion. 2. all nodes with any resourceVersion. 3. all ConfigMaps in all namespaces with any resourceVersion and label `app=antrea`. |
```

Comment:
@wenyingd : we collect all Pods, etc. in the cluster? Why? I feel we should not do this. Not good to collect all workload information from customer clusters either.

Reply:
I feel this is wrong. But let us discuss and handle that separately. @tnqn @edwardbadboy